tests/gnrc_dhcpv6_client_6lbr/tests-as-root/01-run.py

```diff
@@ -14,7 +14,7 @@
 
 from scapy.all import AsyncSniffer, sendp, Ether, IPv6, UDP
 from scapy.all import DHCP6_Solicit, DHCP6_Advertise, DHCP6_Request, DHCP6_Reply
-from scapy.all import DHCP6OptClientId, DHCP6OptServerId, DHCP6OptIA_PD, DHCP6OptUnknown
+from scapy.all import DHCP6OptClientId, DHCP6OptServerId, DHCP6OptIA_PD, DHCP6OptMudUrl
```

Comment:
This should go into its own commit at least, but since it is _completely_ unrelated, I would even prefer its own PR. This might also be very dependent on the scapy version.

Reply:
I already made a separate commit for this change (5e4b07de89) but you are right that this not related to the actual content of the PR. I will move the commit to separate branch and open a new PR :) As the Option has only be included in the very last version of scapy should there maybe be a hint to which version is required to to actually perform the test?

Reply:
Maybe some way of keeping it optional for now would be good. I.e. Import `DHCP6OptMudUrl` but stick to the old scheme on `ImportError`.

Reply:
I've opened #16593 now which should incorporate your suggestions :) In turn I will remove 5e4b07d from this PR.